presto-orc/pom.xml

```diff
@@ -82,6 +82,11 @@
             <artifactId>presto-common</artifactId>
         </dependency>
 
+        <dependency>
```

Comment:
What is the practical motivation from moving these classes from `presto-orc`?  I don't think this is a good idea to have `presto-orc` depend on `presto-hive-common` and `hadoop-apache2`. What do you think @highker ?

Have you consider keeping these classes in `presto-orc` for now? Although I can see that this might not be the best approach for dependency encapsulation, it might be a lesser evil.

Reply:
Correct. presto-orc should have minimal dependency on presto internal concept. Even for presto-have-common, it's good to keep the dependency small.

Reply:
@arhimondr @highker If presto-hive-common should have minimal dependency on presto internal concept, then the best place to put DataSink concept should be presto-hive,  then both presto-orc and presto-presto-hive-common and PageFile classes are happy.

Reply:
`presto-orc` depends on `DataSink`. It looks like it should be visible in there. Is there a good reason why not to keep `DataSink` in the `presto-orc` module?

Reply:
That's right,  my motivation is to remove dependency between `presto-orc` and `PageFile`